lib/modules/manager/argocd/types.ts

```diff
@@ -6,12 +6,14 @@ export interface ApplicationSource {
   chart?: string;
   repoURL: string;
   targetRevision: string;
+  ref?: string;
 }
 
 export interface Application extends KubernetesResource {
   kind: 'Application';
   spec: {
-    source: ApplicationSource;
+    source?: ApplicationSource;
+    sources?: Array<ApplicationSource>;
```

Comment:
```suggestion
    sources?: ApplicationSource[];
```
use short array syntax